modules/template/elements/BaseTemplateElementContent.php

```diff
@@ -220,10 +143,10 @@ protected function createCopy()
     public function scenarios()
     {
         return [
-            self::SCENARIO_DEFAULT => ['fileList', 'definitionPostData', 'fields'],
-            self::SCENARIO_CREATE => ['fileList', 'definitionPostData', 'fields'],
-            self::SCENARIO_EDIT_ADMIN => ['fileList', 'definitionPostData', 'fields'],
-            self::SCENARIO_EDIT => ['fileList', 'definitionPostData', 'fields'],
+            self::SCENARIO_DEFAULT => ['fileList', 'definitionPostData', 'dynAttributes'],
```

Comment:
@yurabakhtin Is it really necessary to mark 'dynAttributes' as 'safe attributes'? Isn't that only necessary for attributes that are read in via 'load', for example, or for user input?
It is only used internally and shouldn't be necessary.

Reply:
@luke- Yes, you are right, I have removed that [here](https://github.com/humhub/custom-pages/pull/367/commits/c666ba0c409091dd204767402a6d3f9c6959b94a).